the-event-loop/interval.js

```diff
@@ -0,0 +1,22 @@
+let currentCount = 3;
+
+setInterval(
+  function () {
+    if (currentCount === 3) {
+      console.log('3');
+    }
+
+    if (currentCount === 2) {
+      console.log('2');
+    }
+
+    if (currentCount === 1) {
```

Comment:
This sequence of `if` statements works, but if you decide to start at 10 instead of 3, it would require making a lot of changes to this code. Can you do this without all the `if` statements?